src/Components/CommandPrompt/CommandPrompt.module.sass

```
@@ -0,0 +1,73 @@
+@use "../../Assets/colors"
+@use "../../Assets/dimensions"
+@use "../../Assets/transitions"
+@use "../../Assets/shadows"
+
+$lineSize: 28px
+$sideMargin: 16px
+
+.overlay
+    display: none
+    &.shown
+        display: block
+
+    position: fixed
+    left: 0
+    right: 0
+    top: 0
+    bottom: 0
+    width: 100%
+    height: 100%
+    z-index: 1000
+    background-color: rgba(black, 0.2)
+
+    .commandPrompt
+        width: 100%
+        height: 100%
+        max-width: 800px
+        max-height: 400px
+
+        border-radius: dimensions.$border-radius-large
+        background-color: colors.$background-primary
+        @include shadows.default-shadow()
+
+        .inputBar
+            padding: $sideMargin
+            border-bottom: solid 1px colors.$background-tertiary
+
+            .inputField
+                border: none
+                margin-left: 8px
+                width: 100%
+                height: $lineSize
+                font-size: calc(#{$lineSize} * 2/3)
+                background-color: colors.$background-primary
+                &:focus
+                    outline: none
+
+        .matchedCommand
+            list-style: none
+            padding: $sideMargin
+            height: $lineSize
+            font-size: calc(#{$lineSize} * 2/3)
+
+            &.selectedCommand
+                background-color: colors.$accent
+                color: colors.$background-primary
+
+                .matchedCommandPart
+                    color: colors.$background-secondary
+                    border-bottom: solid 2px colors.$background-secondary
+
+            &:hover:not(.selectedCommand)
+                background-color: colors.$background-secondary
+
+            .commandName
+                display: inline-block
+
+            .matchedCommandFirstPart
+                margin-left: 12px
+
+            .matchedCommandPart
+                color: colors.$accent
+                font-weight: bold
```

```suggestion
                font-weight: bold
                
```
Add newline at the end of file